sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientMockTests.cs

```diff
@@ -660,5 +661,160 @@ public async Task DeserializeTextAnalyticsError()
             Assert.AreEqual(TextAnalyticsErrorCode.InvalidDocument, resultError.Error.ErrorCode.ToString());
             Assert.AreEqual("Document text is empty.", resultError.Error.Message);
         }
+
+        #region Analyze
+        [Test]
+        public async Task AnalyzeOperationKeyPhrasesWithDisableServiceLogs()
+        {
+            var mockResponse = new MockResponse(202);
+            mockResponse.AddHeader(new HttpHeader("Operation-Location", "something/jobs/2a96a91f-7edf-4931-a880-3fdee1d56f15"));
+
+            var mockTransport = new MockTransport(new[] { mockResponse, mockResponse });
+            var client = CreateTestClient(mockTransport);
+
+            var documents = new List<string>
+            {
+                "Elon Musk is the CEO of SpaceX and Tesla."
+            };
+
+            var options = new ExtractKeyPhrasesOptions()
+            {
+                DisableServiceLogs = true
+            };
+
+            TextAnalyticsActions batchActions = new TextAnalyticsActions()
+            {
+                ExtractKeyPhrasesOptions = new List<ExtractKeyPhrasesOptions>() { options },
+            };
+
+            await client.StartAnalyzeBatchActionsAsync(documents, batchActions);
+
+            var content = mockTransport.Requests.Single().Content;
+            var stream = new MemoryStream();
+            await content.WriteToAsync(stream, default);
+            stream.Position = 0;
+            var streamReader = new StreamReader(stream);
```

Comment:
`MemoryStream` and `StreamReader` implement the IDisposable interface, so I suggest adding the `using` keyword for proper disposal:
```cs
using var stream = new MemoryStream();
using var streamReader = new StreamReader(stream);
```